server/core_notification.go

```diff
@@ -322,3 +325,69 @@ SELECT
 
 	return nil
 }
+
+func NotificationsGetId(ctx context.Context, logger *zap.Logger, db *sql.DB, ids ...string) ([]*runtime.Notification, error) {
```

Comment:
We should include the user ID and caller ID parameters so we can optionally enforce access controls.